File: modul 5/c/PRAK501-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d){
  int left, right, max;
  (a>b) ? (left = a) : (left = b);
  (c>d) ? (right = c) : (right = d);
  (left>right) ? (max = left) : (max = right);
  return max;
}

int main(){
  int a,b,c,d;
  scanf("%d %d %d %d", &a, &b, &c, &d);
  int hasil = MaxBilangan(a,b,c,d);
  printf("%d\n", hasil);
  return 0;
}

File: modul 5/python/PRAK501-2410817110008-MuhammadAzmaAlFaqih.py
a,b,c,d = map(int, input().split())

def MaxBilangan(a,b,c,d):
    left = a if b<a else b
    right = c if d<c else d
    return left if right<left else right

print(MaxBilangan(a,b,c,d))